schemas/rule.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used by observers and agents to trigger events and/or actions.
+  description: >
+   Rule fields are used to capture the specifics of any observer or agent rules that are associated with a given action or event.
+   Typical sources of rule data might include any of the following: acl enabled network access equipment, network admission control 
+   platforms, network or host IDS/IPS, network firewalls, web access firewalls, url filters, etc.   
```

Comment:
consider changing: "platforms, network or host IDS/IPS, network firewalls, web access firewalls, url filters, etc." to "platforms, network or host IDS/IPS, network firewalls, web access firewalls, url filters, endpoint detection and response (EDR) systems, etc."

Reply:
done